prime-router/src/main/kotlin/cli/TestReportStream.kt

```diff
@@ -78,10 +80,25 @@ Examples:
     private val run by option(
         "--run",
         metavar = "test1,test2",
-        help = """Specify set of tests to run.   Default is to run all if not specified.
-            Allowed tests are: ${coolTestList.joinToString(",")}
+        help = """Specify names of tests to run.   Default is to run all smoke tests if not specified.
+            Use --list to see a list of all the tests.
        """
     )
+    private val itemsDefault: Int = 5
+    private val items by option(
+        "--items",
+        metavar = "<int>",
+        help = "For tests involving fake data, 'items' rows **per receiver getting data**. " +
+            "Default is $itemsDefault"
+    ).int().default(itemsDefault)
+
+    private val submitsDefault: Int = 5
+    private val submits by option(
+        "--submits",
+        metavar = "<int>",
+        help = "For tests involving multiple submits, do this many submissions. " +
+            "Default is $submitsDefault"
+    ).int().default(itemsDefault)
```

Comment:
Can now specify the number of files to submit, and number of items per file, for some tests.